tmva/tmva/inc/TMVA/BatchGenerator/RChunkConstructor.hxx

```diff
@@ -0,0 +1,249 @@
+// Author: Dante Niewenhuis, VU Amsterdam 07/2023
+// Author: Kristupas Pranckietis, Vilnius University 05/2024
+// Author: Nopphakorn Subsa-Ard, King Mongkut's University of Technology Thonburi (KMUTT) (TH) 08/2024
+// Author: Vincenzo Eduardo Padulano, CERN 10/2024
```

Comment:
If this file contains new code, then authors of other files do not need to be included